Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG IMAGE_BUILD_NODEJS=launcher.gcr.io/google/nodejs
+ARG IMAGE_BUILD_NODEJS=node:current@sha256:e8e882c692a08878d55ec8ff6c5a4a71b3edca25eda0af4406e2a160d8a93cf2
```

Comment:
We might need to get some help on this one internally or use Prometheus/official node image here.. We can't use launcher.gcr.io/google/nodejs anymore.

Reply:
@bernot-dev found us-central1-docker.pkg.dev/serverless-runtimes/google-22/runtimes/nodejs22 which seems to well maintained and fits our needs!